.github/workflows/security-workflow.yml

```
@@ -0,0 +1,54 @@
+name: Security Workflow
+
+on: [push, workflow_dispatch]
+
+jobs:
+  static-analysis:
+    name: "Static Analysis"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+
+      - name: Run Gitleaks
+        id: gitleaks
+        uses: olxbr/gitleaks-action@main
+
+      - name: Submit Gitleaks metrics
```

Same suggestion from the other PR. https://github.com/olxbr/nginx-images/pull/41#discussion_r750577018
```suggestion
      - name: Submit Gitleaks metrics
        if: always()
```